app/assets/javascripts/components/actions/statuses.jsx

```diff
@@ -28,7 +28,6 @@ export function fetchStatus(id) {
     const skipLoading = getState().getIn(['statuses', id], null) !== null;
 
     dispatch(fetchContext(id));
-    dispatch(fetchStatusCard(id));
```

Comment:
@Gargron If a status was deleted in the database, but existed on a frontend and someone clicked on it, then you would get 2 error flashes, when you should be getting 1.